tensorrt_llm/_torch/disaggregation/resource/kv_extractor.py

```
@@ -313,11 +333,63 @@ def _build_layer_group_for_v2_mamba(
+                f"{sorted(missing_layers)}"
+            )
+        pool_idx = len(pools)
+        pools.append(pool)
+        entries = np.array(
+            [
+                (
+                    global_to_local[layer_id],
+                    int(state.data_ptr()) - pool.base_address,
+                    pool.slot_bytes,
+                )
+                for layer_id, state in zip(layer_ids, states)
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)